t/exit.t

```diff
@@ -10,7 +10,7 @@ log_level('warn');
 repeat_each(120);
 #repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 5);
+plan tests => repeat_each() * (blocks() * 5) - (repeat_each() * 2);
```

Comment:
IIRC (this is a bit old of a PR now), that is because TEST 6 and TEST 7 are each performing one less test than the other tests in this file. Hence `- repeat_each() * 2` does the trick.